### ISSUE TITLE
[UI] Create a experiment with default configuration

### ISSUE THREAD

**Issue by oidacra** [dotCMS/core#22983]
### Is your feature request related to a problem? Please describe.
[Experiment] - Configuration View of an experiment

### Describe the solution you'd like
Create all the components to show the default configuration of a recently created experiment.

### Describe alternatives you've considered
Final view
![Create Experiments - empty](https://user-images.githubusercontent.com/1909643/190221381-8eddbccf-d454-43ac-b5cf-79c6a884c65c.png)

### Additional context
Only SHOW the default configuration.

### Acceptance criteria
- Create Details and Reports containers
![Screen Shot 2022-09-15 at 11.27.58 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/e759b345-949f-4256-953f-805b4224571f)

- Create a component to show the status of the experiment
![Screen Shot 2022-09-15 at 11.28.35 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/b56a66cc-b335-4b3b-a8ff-fc37d58e6d83)

- Create a component to show the steps finished
 ![Screen Shot 2022-09-15 at 11.29.13 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/8e8a69e2-be87-429c-bd8a-ae108aed03a0)

- Card of Variation loaded with the original variation (Reuse)
![Screen Shot 2022-09-15 at 11.31.56 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/20519fbf-c1e0-448c-a9e1-749e66ad49f7)

- Card of 'Set goal' empty (Reuse)
![Screen Shot 2022-09-15 at 11.33.13 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/eefb76ec-b586-4494-9348-2f01fa3d1542)

- Card of 'Set target' empty (Reuse)
![Screen Shot 2022-09-15 at 11.34.25 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/c5a0212b-8547-4c42-adae-722114dc20cd)

- Card of 'Set traffic' with the defaults (Reuse)
![Screen Shot 2022-09-15 at 11.35.15 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/c4577b6b-9bb7-4bff-ace5-c9d9f2e1dccb)

- Card of 'Set Scheduling' empty (Reuse)
![Screen Shot 2022-09-15 at 11.36.13 AM.png](https://images.zenhubusercontent.com/621e3b2d355dc73757435ad6/2f77b0f5-5de3-4908-a19a-b68b903bb59b)

### Experiment Object with default config.
```
"entity": [
        {
            "archived": false,
            "createdBy": "dotcms.org.1",
            "creationDate": "2022-09-13T13:53:32.092Z",
            "description": "my experiment description",
            "id": "230fd8ef-b5b2-4a84-9aae-783919dddf56",
            "lastModifiedBy": "dotcms.org.1",
            "modDate": "2022-09-13T13:53:32.103Z",
            "name": "my experiment",
            "pageId": "a9f30020-54ef-494e-92ed-645e757171c2",
            "scheduling": null,
            "status": "DRAFT",
            "trafficAllocation": 100.0,
            "trafficProportion": {
                "percentages": {},
                "type": "SPLIT_EVENLY"
            }
        }
    ],
```

**Comment by oidacra** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Falcon-6176e9d907eb0000106ee8e1/issues/dotCMS/core/22983?planning-poker) with Zenhub @alfredo-dotcms @rjvelazco @fmontes @hmoreras

**Comment by dsilvam** [comment]
Passed Internal QA: After creating an Experiment, it is showing all the sections. 

<img width="1722" alt="Screenshot 2022-11-08 at 15 42 45" src="https://user-images.githubusercontent.com/1576794/200682164-f04eaec8-b578-4c91-98b5-e05ff84a0d83.png">

**Comment by bryanboza** [comment]
Fixed, tested on release-23.01 // Docker // FF
